### ISSUE TITLE
Option to customize name in the footer

### ISSUE THREAD

**Issue by cuducos** [maaslalani/slides#90]
**Is your feature request related to a problem?**

Not a problem, just a new use case. Usually I don't use my (full) name in presentations, I tend to use my social network user handle instead — in my case, `@cuducos` instead of `Eduardo Cuducos`.

**Describe the solution you'd like**

I can open a PR to add a flag to overwrite the user name when used, e.g.:

```console
# slides exmaple.md --name "@cuducos"
```

This would use `@cuducos` in the “footer” instead of the full name of the logged user. This logic can be wrapped in `cmd/root.cmd` using Cobra.

**Describe alternatives you've considered**

Maybe we could use another naming, such as `--footer` — what do you think?

**Comment by maaslalani** [comment]
I proposed a solution to solve both this issue and #87 

With https://github.com/maaslalani/slides/issues/87#issuecomment-893434901

Something like the following at the top of the slides.

```
---
author: @cuducos
date: 1970-01-01
---
```

I think this approach is nicer than having to add CLI flags everytime.

**Comment by cuducos** [comment]
Awesome! Closes #91
